### ISSUE TITLE
Giving account money ox_inventory

### ISSUE THREAD

**Issue by og23rd** [esx-framework/esx-legacy#103]
I have a custom account setted up in es_extended/config.lua and when i do the command "giveaccountmoney" it gives me the account money but not the item in ox_inventory. When i do the command "giveitem [account name]" (already created the item and configured in ox_inventory/modules/bridge/server.lua as server account)  it gives me the item and the account money...

**Comment by thelindat** [comment]
https://github.com/esx-framework/esx-legacy/blob/main/%5Besx%5D/es_extended/server/classes/player.lua#L194-L244
```lua
if Inventory and (accountName == 'money' or accountName == 'black_money') then
```

I'll probably have to expose the the table for defining which accounts are physical items; checking if the account exists as an item would also work.

**Comment by og23rd** [comment]
or what do i need to modify in that line to make it work?

**Comment by og23rd** [comment]
also when i do the command "setaccountmoney" it removes te account money but keep the items xDD that happens for all the accounts

**Comment by og23rd** [comment]
> also when i do the command "setaccountmoney 1 money 0" for example it removes te account money but keep the items xDD that happens for all the accounts

**Comment by 0xDEMXN** [comment]
This should be solved with #113
